src/index.html

```diff
@@ -22,6 +22,373 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="logo"
+          src="/src/images/logo.png"
+          alt="logo"
+        />
```

Comment:
The `href` attribute in the logo link should point to the root (`/`) rather than `index.html` to ensure the link always directs to the homepage regardless of the current URL.